tap_pendo/streams.py

```diff
@@ -921,6 +806,14 @@ class VisitorHistory(Stream):
     replication_key = "modified_ts"
     key_properties = ['visitor_id']
     DATE_WINDOW_SIZE = 1
+    headers = {
+        'content-type': 'application/x-www-form-urlencoded'
+    }
+    params = {
+        "starttime": "start_time"
+    }
+    endpoint = Endpoints(
+        "/api/v1/visitor/{visitorId}/history", "GET", headers, params)
```

Comment:
Add comments to this code change.
Also, can you please explain, how would the 'VisitorId' parameter be assigned a value?

Reply:
Done.
The get_url function of the Endpoints class will be used, and in that function this `visitorId` will be formatted and replaced with the original one